src/Sound.cpp

```diff
@@ -41,11 +41,6 @@ bool Sound::init() {
     return true;
 }
 
-void Sound::createSound(const char *pFile) {
-    
-    m_pSystem->createStream(pFile, FMOD_DEFAULT, 0, &newSound);
-}
-
```

Comment:
Every time createSound was called it stores part of the song in memory, but doesn't remove anything from memory unless the previous sound was being played. It would cause a memory leak (a small one) when songs are opened.